server/controllers/users.js

```diff
@@ -1,12 +1,10 @@
 const mongoose = require('mongoose')
 const User = mongoose.model('User')
-const consola = require('consola')
 
 module.exports = {
   findUserByEmail: async email => {
-    const user = await User.find({ email }).limit(1)
-    if (user) return user
-    consola.log('User not found')
+    const user = await User.findOne({ email })
```

Comment:
No catch state ment if user is not found

Reply:
Will return null, won't error

Reply:
Doesn't error if no user